content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -167,10 +167,8 @@ and Linux>> instructions above.
 If you have some experience with Docker and you wish or need to access the
 Jenkins/Blue Ocean Docker container through a terminal/command prompt using the
 link:https://docs.docker.com/engine/reference/commandline/container_exec/[`docker container exec`]
-command, you can add an option like `--name jenkins-tutorials` (with the
-link:https://docs.docker.com/engine/reference/commandline/container_run/[`docker container run`]
-above), which would give the Jenkins/Blue Ocean Docker container the name
-"jenkins-tutorials".
+command, you can add an option like `--name jenkins-tutorial` to the `docker exec` command.
+Taht will access the the Jenkins Docker container named "jenkins-tutorial".
```

Comment:
Typo on that

Reply:
> Typo on that

Wow!  So many errors in  a single line of text.  Thanks!  I really should not write documentation after 10:00 PM.  Fixed in afdd8441d1db387511cbb5383ea921a8bb09d30d